SharedCode/src/commonMain/kotlin/ApplicationPresenter.kt

```diff
@@ -81,13 +75,12 @@ class ApplicationPresenter: ApplicationContract.Presenter() {
         val journeyTime: TimeSpan = arrivalDateTime - departureDateTime
         val journeyTimeMinutes: String = "${journeyTime.minutes}m"
         val trainOperator = journeyDetails.primaryTrainOperator.name
-        lateinit var price: String
-                price = try{
+        var price = try{
```

Comment:
Should have a space before the bracket here, and the indentation here looks a bit funny - Android Studio has an auto-formatter that you can run by doing Cmd-Option-L which can be helpful for sorting these things out, though it's often a good idea to run it while selecting a specific block of code so you can see exactly what it's doing and catch it if it has any odd default settings